### ISSUE TITLE
Installer fails on Windows - unable to create shortcut

### ISSUE THREAD

**Issue by nielsvanvelzen** [jellyfin/jellyfin-media-player#27]
I tried installing the latest release (JellyfinMediaPlayer-1.3.0-windows-x64.exe) on my Windows 10 machine but the installer was unable to create a shortcut. Same issue when running the installer as administrator.

![image](https://user-images.githubusercontent.com/2305178/114569425-3e952300-9c75-11eb-8cfb-a7a40ae79513.png)

I changed the install directory to `D:\Programs\Jellyfin Media Player` but I don't think that should cause this issue.

**Comment by iwalton3** [comment]
Did the installer ask you to elevate with UAC?

**Comment by iwalton3** [comment]
I haven't had any luck reproducing this. I created a VeraCrypt volume (to simulate another drive letter) in my VM and it installed there fine. It asked me to elevate using UAC and installed just fine.

**Comment by nielsvanvelzen** [comment]
> Did the installer ask you to elevate with UAC?

If I don't run the installer as administrator it asks for it.

> I haven't had any luck reproducing this. I created a VeraCrypt volume (to simulate another drive letter) in my VM and it installed there fine. It asked me to elevate using UAC and installed just fine.

Weird, I never had this issue happen with other programs. I checked the permissions of the target folder and it looks normal.

**Comment by nielsvanvelzen** [comment]
After further investigation it appears that the permissions of my "Start Menu" folder are corrupt. Weirdly enough, other software still installs fine (and is able to create shortcuts). From what I've read on other websites the only fix is to reinstall Windows (which I'm not wanting to do right now).

Do you think it's possible to create a portable Windows release as a workaround?

**Comment by iwalton3** [comment]
The application needs to install some vcredist dependencies so a portable one is unlikely to work well. You might want to try adjusting the permissions of the start menu with the advanced settings.

**Comment by nielsvanvelzen** [comment]
So I was finally able to fix the issue, I had to (manually) reset the permissions for the following folders:

- `C:\ProgramData\Microsoft\Windows\Start Menu`
- `C:\ProgramData\Microsoft\Windows\Start Menu\Programs`

and now it works again. For some reason applying the permissions to the subfolders didn't work so I had to apply it to both manually. I took the permissions from my program files folder. In the end the issue was all on my system. Thanks for the help!
